rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block title %}
+CSDT Classroom Guide
+{% endblock %}
+
+{% block content %}
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-10 col-md-offset-1" style="font-size: 1.3em;">
+            <h1>CSDT Classroom Creation Guide</h1>
+            <p>
+                Thank you for your interest in signing your classroom up to use our software, Culturally Situated Design Tools. We have programmed a "classrooms" app on our website to facilitate teachers who are running a classroom of students, to address their needs as facilitators. CSDTs are set up to simulate a cloud computing experience. Each user can save their program to the cloud, which shows up under their profile even if they're not in a classroom. The purpose of classrooms is to organize this cloud computing into a sort of gallery or boutique for all the students in the classroom, including the teacher. With a classroom, students can create and save their programs, then submit them for approval by the teacher or teachers. The teacher approves projects using a visual representation of the program, to verify that it is school appropriate, and then all approved projects are added to the pool of works created by the classroom. This pool of shared works can be viewed by other students, commented on, and even remixed by other students using the other's person's source code. Classrooms allow for sharing between students and at-a-glance observation of who is creating what, providing motivation and teamwork. For the teacher, it permits an easy source to view who is keeping up with the tutorials, who is struggling, who is completing assignments, who is succeeding, and who is creating genuinely interesting works of art.
```

Comment:
Teachers != students: "boutique for all the students in the classroom" -> "boutique for all the members of the classroom"

Doing more than tutorials "who is keeping up with the tutorials" -> "Who is keeping up with their work"